docs/intermediate-modding/custom-configs.md

```diff
@@ -0,0 +1,86 @@
+
+---
+prev: false
+next: false
+description: An intermediate overview of how to impliment custom configs for your Lethal Company mods.
+---
+
+# Custom Configs
+
+::: warning
+**This is an intermediate article. While this introduces some C# concepts, it is recommended to understand C# and the basics of modding this game <i>before</i> reading this article.**
+:::
+
+::: info
+This tutorial is taken and adapted from the [BepInEx Configuration Documentation](https://docs.bepinex.dev/articles/dev_guide/plugin_tutorial/4_configuration.html). For more resources refer to that.
+:::
+
+## Creating Config Entries
+Create a config class and add entries for any variables that you want to be configurable.
+
+```cs
+public class Config
+{
+    public static ConfigEntry<string> configGreeting;
+    public static ConfigEntry<bool> configDisplayGreeting;
+
+    // ...
+}
+```
+
+Then we can start binding our config entries to the fields we just created inside of a class constructor.
+
+```cs
+public Config()
```

Comment:
In plugin Awake you are calling `MyConfig = new(base.Config);` so it seems you forgot the `ConfigFile cfg` parameter.
Then rename all instances of `Config.Bind` to `cfg.Bind` to avoid errors.